src/store/getters.ts

```diff
@@ -187,23 +193,28 @@ export default {
   },
 
   /**
-   * Is True if business is in Limited Restoration state, ie
-   * the last filing to change state was a Restoration filing.
+   * True if the business is in limited restoration, ie, the state filing is a
+   * Limited Restoration filing or Limited Restoration Extension filing.
    */
-  isEntityInLimitedRestoration (_state: StateIF, _getters, rootState): boolean {
+  isInLimitedRestoration (_state: StateIF, _getters: any, rootState: any): boolean {
     const stateFiling = rootState.stateFiling // may be null
 
-    return (stateFiling?.hasOwnProperty(FilingTypes.RESTORATION))
+    return (
+      !!stateFiling && (
+        EnumUtilities.isTypeRestorationLimited(stateFiling) ||
+        EnumUtilities.isTypeRestorationExtension(stateFiling)
+      )
+    )
   },
 
   /**
-   * Is True if business is in Authorized to Continue Out state, ie
-   * the last filing to change state was a Consent to Continue Out filing.
+   * True if the business is authorized to continue out, ie, the state filing is a
+   * Consent to Continuation Out filing.
    */
-  isAuthorizedToContinueOut (_state: StateIF, _getters, rootState): boolean {
+  isAuthorizedToContinueOut (_state: StateIF, _getters: any, rootState: any): boolean {
     const stateFiling = rootState.stateFiling // may be null
 
-    return (stateFiling?.hasOwnProperty(FilingTypes.CONSENT_CONTINUATION_OUT))
+    return (stateFiling?.hasOwnProperty(FilingTypes.CONSENT_CONTINUATION_OUT) || false)
```

Comment:
Isn't it possible to access the `stateFiling` via `state` and not `rootState`?  If at all possible, I think we should avoid using `rootState` unless there are no other options.

Reply:
Yes, I think `stateFiling` is in this store's module, so I should be able to use state. Stand by...

Reply:
I've update this. I've decided to use `getters` to get the state filing, so that's there only 1 place where the state is fetched.

Reply:
I ❤️ it!  Beautiful solution.